app/src/main/java/org/mozilla/focus/home/NavigationOverlayFragment.kt

```diff
@@ -106,6 +93,37 @@ class NavigationOverlayFragment : Fragment() {
         }
     }
 
+    private fun setOnClickListeners(overlay: View) {
+        // We remove the click listeners when dismissing the overlay
+        // so that clicking them doesn't restart the animation.
+        fun removeClickListeners() {
+            overlay.semiOpaqueBackground.setOnClickListener(null)
+            with(overlay.homeTiles) {
+                onTileClicked = {}
+                homeTileLongClickListener = null
```

Comment:
Any particular reason why one of these is removed using null, and the other with a blank listener?  If so, a comment would help.